### DISCUSSION TITLE
Connecting T2S3 with DHT11

### DISCUSSION THREAD

**Discussion by CrazyTolstoy** [meshtastic/firmware#4299] [Q&A] [answered]
Hello friends,
Recently, I started with some DIY projects with controller like esp8266 and sensors like soil moisture sensor and dht11/22 sensors, L298N H-bridge for dc motor controlling, or 5v relays for light controlling over web app. Next thing I wanted to do is to make project with LoRa technology, so my setups don't need to be limited to area with wifi only. I bought two LILYGO® T3S3 ESP32-S3 LoRa SX1262 2.4G Development Board WiFi Bluetooth Wireless Module 0.96 Inch OLED Display Type-C (https://www.banggood.com/LILYGO-T3S3-ESP32-S3-LoRa-SX1262-or-SX1276-SX1280-2_4G-Development-Board-WiFi-Bluetooth-Wireless-Module-0_96-Inch-OLED-Display-Type-C-p-1997240.html) and installed firmware using https://flasher.meshtastic.org/, and it worked, and I succeded to send messages from one board to another using https://client.meshtastic.org/. But now I want to connect my board with dht11 sensor, and send humidity and temperature values with LoRa to my second board, but I just cant do anything. I even tried most simple code to install using arduino ide, just to show some messages in serial monitor, but even though it writes me I successfully installed it, nothing happens. I tried also to put board in boot (turned it off, pressed boot btn, turned it on, and after 3 sec release boot btn).
I'm totally new in this and I would really appreciate your help!
Thanks in advance! <3 
![image](https://github.com/user-attachments/assets/237779c0-cfe9-444c-bf9b-4df00e45cbb7)
![image](https://github.com/user-attachments/assets/31c849ba-a899-4e82-a767-394b9e717443)

**Comment by garthvh** [comment]
The dht11 is not supported, you need a i2c sensor

**Reply by CrazyTolstoy** [reply]
The main thing is I can't install any custom code, I tried simple one with just message printing in serial monitor on loop, but I got nothing.

**Reply by CrazyTolstoy** [reply, marked as answer]
I found the problem, I had to set usb cdc on boot to enabled...
